spec/domain/operations/premium_credits/find_aptc_spec.rb

```diff
@@ -174,7 +178,7 @@
             end_on: TimeKeeper.date_of_record.end_of_year,
             assistance_year: TimeKeeper.date_of_record.year,
             member_ids: family.family_members.map(&:id).map(&:to_s),
-            tax_household_id: tax_household.id
+            tax_household_id: inactive_tax_household.id
           )
```

Comment:
Can you add a new context with another example by building eligibility determination with inactive tax household in it